pkg/ccl/changefeedccl/testfeed_test.go

```diff
@@ -906,6 +925,36 @@ func (f *cloudFeedFactory) Feed(
 		)
 	}
 
+	parquetPossible := true
```

Comment:
Let's add some commentary on what's going on.
(i.e. determine if we can enable parquet output if the changefeed is compatible with parquet format).